website/functions/get-discourse-posts.js

```diff
@@ -0,0 +1,100 @@
+const axios = require('axios')
+
+async function getDiscoursePosts({ body }) {
+  const { DISCOURSE_API_KEY , DISCOURSE_USER } = process.env
+
+  try {
+    // Set API endpoint and headers
+    let discourse_endpoint = `https://discourse.getdbt.com`
+    let headers = {
+      'Accept': 'application/json',
+      'Api-Key': DISCOURSE_API_KEY,
+      'Api-Username': DISCOURSE_USER,
+    }
+
+    const query = buildQueryString(body)
+    if(!query) throw new Error('Unable to build query string.')
+    
+    // Get topics from Discourse
+    let { data: { topics } } = await axios.get(`${discourse_endpoint}/search?q=${query}`, { headers })
+    
+    if(!topics)
+      throw new Error('Unable to get results from api request.')
+
+    // Return posts 
+    return await returnResponse(200, topics)
+  } catch(err) {
+    // Log and return the error
+    console.log('err', err)
+    return await returnResponse(500, { error: 'Unable to get events from Discourse.'})
```

Comment:
I assume this is copy-paste from the events feed
```suggestion
    return await returnResponse(500, { error: 'Unable to get topics from Discourse.'})
```